### ISSUE TITLE
Awesome concept which simply doesnt work for me.

### ISSUE THREAD

**Issue by Latzi** [yueruchen/Pedestrian-Synthesis-GAN#37]
Apart from some issues in the code 

networks.py : change _id to device in line 91
networks.py : change the division (/) to int division (//) lines 451 452.
base_model.py : change device_id to device in line 47.
pix2pix_model.py : change from self.XXX.data[0] to self.XXX.cpu().data in all lines 189 to 196

The model trains okay but the results are really bad. I am not sure what I am doing wrong ? I used a dataset of roughly similar looking images, pedestrians on sidewalks , size 256 x 256, number of images 500. Training completes the loss graphs look okay all is looking good but when running test.py to generate pedestrians in images with similar backgrounds but no pedestrians I have really bad results. I trained the model with only 6 images same background but different pedestraisn and the model once again trains really well but when testing the model on the vers same images it was trained on it cannot recreate the training images. I would have expected that those images would be learned and recterated but it cannot. I tried experimenting with hyperparameters, learning rates, weighting of the lossess , UNET depth , you name it I tried it but unfortunatley I still can't get it to work. I think the ide is awesome and sound and it should work but not for me :-( . Can you please point me in some direction on what I might be doing wrong ?
